oioioi/mp/templates/mp/registration.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base-with-menu.html" %}
```

Comment:
If we depend on `oioioi.participant` we should be able to use its templates. This is again 1:1 copied file from oioioi/oioioi/participants/templates/participants/registration.html